### ISSUE TITLE
Ternary operator causing template problems

### ISSUE THREAD

**Issue by brighttank** [statamic/cms#1170]
When using the ternary operator in a template the next tag-block is enclosed in apostrophes.

For example if I have this in my layout.

```
<div>
    I want to use a ternary: {{ is_entry ? "ok" : "no" }}
</div>
{{ partial:_message }}
```

The outputted html looks like this:

```
<div>
    I want to use a ternary: ok
</div>
'This is a message.'
```

Where those apostrophes are not coming from the partial. Same happens if I follow the ternary with a {{ nav }} tag. The output of the nav is enclosed in apostrophes.

If I use the standard {{ if }} {{ else }} {{ /if }} then everything works as expected.

**Comment by jackmcdade** [comment]
I too have run into this. I thought I had an open issue but in fact did not. Thanks!

**Comment by jackmcdade** [comment]
Fixed in fa8be6bf. Thanks!
